css/style.css

```diff
@@ -0,0 +1,551 @@
+*{margin: 0; padding: 0; box-sizing: border-box; list-style: none; text-decoration: none; font-family: roboto;}
```

Comment:
@lakshmikhabbas For CSS try to add scss linter rules here. Is there any reason for selecting CSS over SCSS?